yarn-project/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM --platform=linux/amd64 aztecprotocol/bb.js as bb.js
 FROM --platform=linux/amd64 aztecprotocol/noir-packages as noir-packages
 FROM --platform=linux/amd64 aztecprotocol/l1-contracts as contracts
 FROM --platform=linux/amd64 aztecprotocol/noir-projects as noir-projects
-FROM --platform=linux/amd64 aztecprotocol/noir as noir
+FROM aztecprotocol/noir as noir
 FROM --platform=linux/amd64 aztecprotocol/barretenberg-x86_64-linux-clang as barretenberg
```

Comment:
Shouldn't we remove the linux/amd64 from barretenberg as well, since we're also copying the binary `/usr/src/barretenberg/cpp/build/bin/bb` a few lines below?